dotCMS/src/main/java/com/dotcms/auth/providers/jwt/beans/UserToken.java

```diff
@@ -24,18 +24,19 @@
     private final String issuer;
     private final Date modificationDate;
     private final Date expiresDate;
+    private final String skinId;
     private final ImmutableMap<String,Object> claims;
 
     
     public UserToken(final String id, final String subject, final String issuer, final Date modificationDate,
-            Date expiresDate, final Map<String,Object> claims) {
+            Date expiresDate, final Map<String,Object> claims, final String skinId) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'expiresDate' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5164094)